webpack.config.js

```diff
@@ -93,6 +93,10 @@ const config = {
         }),
         new MiniCssExtractPlugin({ filename: 'smartcharts.css' }),
         new StyleLintPlugin(),
+        new CopyWebpackPlugin([
```

Comment:
@brucebinary Can I leave this piece of code here or I should put in under `if (production) {` ?